ui/app/templates/vault/cluster/policy/show.hbs

```diff
@@ -29,14 +29,14 @@
 <Toolbar>
   <ToolbarActions>
     <DownloadButton
-      class="toolbar-link"
+      class="toolbar-button"
+      @color="secondary"
       @filename={{this.model.name}}
       @data={{this.model.policy}}
       @extension={{if (eq this.policyType "acl") this.model.format "sentinel"}}
-    >
-      Download policy
-      <Chevron @isButton={{true}} />
-    </DownloadButton>
+      @icon="chevron-right"
```

Comment:
It's a little odd to me that the icon here is a chevron. It seems to indicate on other toolbars that we will take you to a new page 🤔

Reply:
I agree! I was just keeping with the old pattern but I can update to a download :)